templates/nscd.conf.erb

```diff
@@ -72,8 +72,8 @@
 	shared			hosts		yes
 	max-db-size		hosts		33554432
 
-<% unless @osfamily == 'RedHat' and @operatingsystemrelease.to_f < 6.0 -%>
-	enable-cache		services	<%= @nscd_service_cache %>
+<% unless @osfamily == 'RedHat' and @operatingsystemmajrelease == '5' -%>
```

Comment:
I expect 6 and 7 are the same, configuration probably came from a 6. Should do as an acceptence test if attempting to do it.